rust/schema/annotations.py

```diff
@@ -1520,3 +1520,21 @@ class _:
     todo!()
     ```
     """
+
+class Callable(AstNode):
+    """
+    A callable. Either a `Function` or a `ClosureExpr`.
+    """
+    param_list: optional["ParamList"] | child
+    attrs: list["Attr"] | child
```

Comment:
minor nit, string type references are only required for "forward" references
```suggestion
    param_list: optional[ParamList] | child
    attrs: list[Attr] | child
```